web/autograder_service.go

```diff
@@ -406,17 +406,21 @@ func (s *AutograderService) DeleteGroup(ctx context.Context, in *pb.RecordReques
 }
 
 // GetSubmissions returns the submissions matching the query encoded in the action request.
-// Access policy: Current User if Owner of submission, Teacher of CourseID.
+// Access policy: Current User if Owner of submission (or member of the group in case of group submission), Teacher of CourseID.
```

Comment:
We use comma separate list for disjoint parts of the access policy.
But since this is a long line, I suggest to move it to separate lines, e.g.:
```
// Access policy:
// Current User if Owner of submission,
// Current User is member of group for group submission,
// Teacher of CourseID.
```